duet/views/ad_cost_breakdowns.view.lkml

```diff
@@ -0,0 +1,124 @@
+include: "//looker-hub/revenue/views/ad_cost_breakdowns.view"
+
+view: +ad_cost_breakdowns {
+
+  dimension: campaign_name {
+    sql: ${TABLE}.campaign_name ;;
+    type: string
+  }
+
+  dimension: cost_per_dou_micros {
+    sql: ${TABLE}.cost_per_dou_micros ;;
+    type: number
+    hidden: yes
+  }
+
+  dimension: cost_per_install_micros {
+    sql: ${TABLE}.cost_per_install_micros ;;
+    type: number
+    hidden: yes
+  }
+
+  dimension: cost_per_marketing_ad_click_micros {
+    sql: ${TABLE}.cost_per_marketing_ad_click_micros ;;
+    type: number
+    hidden: yes
+  }
+
+  dimension: cost_per_revenue_generating_ad_click_micros {
+    sql: ${TABLE}.cost_per_revenue_generating_ad_click_micros ;;
+    type: number
+    hidden: yes
+  }
+
+  dimension: ad_conversions {
+    sql: ${TABLE}.ad_conversions ;;
+    type: number
+    hidden: yes
+  }
+
+  measure: measure_ad_conversions {
+    label: "ad_conversions"
+    sql: ${TABLE}.ad_conversions ;;
+    type: sum_distinct
+    sql_distinct_key: ${TABLE}.campaign_name ;;
+  }
+
+  dimension: ad_impressions {
+    sql: ${TABLE}.ad_impressions ;;
+    type: number
+    hidden: yes
+  }
+
+  measure: measure_ad_impressions {
+    label: "ad_impressions"
+    sql: ${TABLE}.ad_impressions ;;
+    type: sum_distinct
+    sql_distinct_key: ${TABLE}.campaign_name ;;
+  }
+
+  dimension: campaign_spend_in_micros {
+    sql: ${TABLE}.campaign_spend_in_micros ;;
+    type: number
+    hidden: yes
+  }
+
+  measure: measure_campaign_spend_in_micros {
+    label: "campaign_spend_in_micros"
+    sql: ${TABLE}.campaign_spend_in_micros ;;
+    type: sum_distinct
+    sql_distinct_key: ${TABLE}.campaign_name ;;
+  }
+
+  dimension: dous {
+    sql: ${TABLE}.dous ;;
+    type: number
+    hidden: yes
+  }
+
+  measure: measure_dous {
+    label: "dous"
+    sql: ${TABLE}.dous ;;
+    type: sum_distinct
+    sql_distinct_key: ${TABLE}.campaign_name ;;
+  }
+
+  dimension: installs {
+    sql: ${TABLE}.installs ;;
+    type: number
+    hidden: yes
+  }
+
+  measure: measure_installs {
+    label: "installs"
+    sql: ${TABLE}.installs ;;
+    type: sum_distinct
+    sql_distinct_key: ${TABLE}.campaign_name ;;
+  }
+
+  dimension: marketing_ad_clicks {
+    sql: ${TABLE}.marketing_ad_clicks ;;
+    type: number
+    hidden: yes
+  }
+
+  measure: measure_marketing_ad_clicks {
+    label: "marketing_ad_clicks"
+    sql: ${TABLE}.marketing_ad_clicks ;;
+    type: sum_distinct
+    sql_distinct_key: ${TABLE}.campaign_name ;;
+  }
+
+  dimension: revenue_generating_ad_clicks {
+    sql: ${TABLE}.revenue_generating_ad_clicks ;;
+    type: number
+    hidden: yes
+  }
+
+  measure: measure_revenue_generating_ad_clicks {
+    label: "revenue_generating_ad_clicks"
+    sql: ${TABLE}.revenue_generating_ad_clicks ;;
+    type: sum_distinct
```

Comment:
Why are these `sum_distinct`? I didn't think the underlying table is denormalized.

Reply:
You are right; that said, since I'm not using these measures at all and since we have clear docs on how to add measures when they are needed, I'm removing them from this PR.